src/ucp/wireup/wireup_cm.c

```diff
@@ -666,37 +662,41 @@ ucp_ep_cm_server_create_connected(ucp_worker_h worker, unsigned ep_init_flags,
                   ucs_sockaddr_str((struct sockaddr*)&conn_request->client_address,
                                    client_addr_str, sizeof(client_addr_str)),
                   conn_request->dev_name);
-        return UCS_ERR_UNREACHABLE;
-    }
-
-    ucp_unpacked_address_for_each(address, remote_addr) {
-        num_remote_ep_addrs += address->num_ep_addrs;
+        uct_listener_reject(conn_request->uct.listener, conn_request->uct_req);
```

Comment:
we shouldn't need reject here since destroying the ep should destroy cm ep and behave as a reject
why needed to add it?

Reply:
I'll remove it since ```destroy the ep should behave as a reject```

Reply:
Done